src/main/java/frc/robot/telemetry/TelemetryIO.java

```diff
@@ -19,6 +19,8 @@ public static class TelemetryIOInputs {
                     new SwerveModuleState(),
                     new SwerveModuleState()
                 };
+        public double accelerationX = 0.0;
+        public double accelerationY = 0.0;
```

Comment:
If I understand this all correctly, there will be 0 acceleration in sim, but it will be measured on the real robot.

Is there some way we can get accel measurements added to simulation? Do we need an IO class for sim telemetry?

Reply:
That would probably have to be the case, and the sim would need supplier from drive train?

Reply:
Or there might be a pigeon sim, I haven't had a chance to look at docs